ClosedXML/Excel/ConditionalFormats/XLConditionalFormats.cs

```diff
@@ -1,13 +1,24 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 
 namespace ClosedXML.Excel
 {
     internal class XLConditionalFormats : IXLConditionalFormats
     {
         private readonly List<IXLConditionalFormat> _conditionalFormats = new List<IXLConditionalFormat>();
+
+        private readonly List<XLConditionalFormatType> _conditionalFormatTypesExcludedFromConsolidation = new List<XLConditionalFormatType>()
```

Comment:
This could be `static`. Maybe better to use `HashSet<>`, @Pankraty ?

Reply:
@ladida28 Please make this static.

Reply:
static is set